### ISSUE TITLE
Removal Of Clojure Related Commands

### ISSUE THREAD

**Issue by tonyjunkes** [framework-one/fw1-commands#7]
Due to what seems to be a growing headache of getting ACF and Lucee (5) to play nice with the cfmljure support in FW/1, I'm going to place this here as a note to remove the commands for creating Clojure controllers and services. When I get to adding the ability to create custom templates (views,controllers,services, etc.), people can just create them like that if they really want them.

**Comment by seancorfield** [comment]
Given where I've gone with CFML + Clojure over the last few months and the EOL of FW/1 for Clojure, I doubt `cfmljure` will get much love going forward so this seems like a wise choice!

**Comment by tonyjunkes** [comment]
I've been following along. Not sure if you saw Micha's most recent update on that ticket but it went about how I figured it would. Hence me doing this.

**Comment by seancorfield** [comment]
Yeah, that went exactly how I expected. Which is why I'd _closed_ that ticket in the first place. I was surprised he reopened it, and I _knew_ he wasn't going to actually solve it properly! Lucee 5 is of no interest to me (because of that bug initially, and now because we're aggressively migrating off CFML -- which is also spurred by Lucee being a dead end for us!).
